ang/civicase/shared/directives/file-uploader.directive.html

```diff
@@ -14,8 +14,11 @@ <h3>{{ts('Drop file here')}}</h3>
       </div>
       <div class="civicase__file-upload-details civicase__file-upload-item pull-right">
         <div class="form-group">
-          <label for="uploadSubject">{{ts('Activity Subject')}}</label>
-          <input type="text" class="form-control" id="uploadSubject" placeholder="" ng-model="activity.subject">
+          <label for="uploadSubject">
+            {{ts('Activity Subject')}}
+            <span class="crm-marker" title="This field is required.">*</span>
```

Comment:
We should use bootstrap here, i.e. `required-mark` class.
For example,
https://github.com/compucorp/uk.co.compucorp.civiawards/blob/a04532a7f4e149211a4d2e48232f8188cacf972f/ang/civiawards/award-creation/directives/basic-details-form.directive.html#L3